cypress/support/elements/clue/ChatPanel.js

```diff
@@ -25,7 +25,7 @@ class ChatPanel{
       return cy.get('[data-testid=comment-card]');
     }
     getCommentTileTypeIcon() {
-      return cy.get('[data-testid=chat-thread-tile-type]').find('svg');
+      return cy.get('[data-testid=chat-thread-tile-type]');
```

Comment:
This test wasn't correctly targeting the SVG, since the` <svg >` tag itself had a data attribute "data-testid=chat-thread-tile-type". Using `.find` only searches a child/descendent element, therefore it kept failing.